### DISCUSSION TITLE
Displaying pickup locations

### DISCUSSION THREAD

**Discussion by BenSampo** [saleor/saleor#16962] [Q&A] [answered]
Without creating a checkout, how can I show a list of available pickup locations to the user in a storefront? Once I have a checkout, this is available on the checkout object.

Querying `Warehouses` requires one of the following permissions: MANAGE_PRODUCTS, MANAGE_ORDERS, MANAGE_SHIPPING which standard users don't have.

**Comment by tomaszlud** [comment, marked as answer]
You can't check pickup locations without permissions. Probably best to make a custom app on the backend to handle location checks and provide the info.
